engine/gamesys/src/gamesys/components/comp_collection_proxy.cpp

```diff
@@ -1043,4 +1093,51 @@ namespace dmGameSystem
      * end
      * ```
      */
+
+    /*# changes the collection for a collection proxy.
+     * 
+     * The collection should be loaded by the collection proxy.
+     * Setting the collection to "nil" will revert it back to the original collection.
+     * 
+     * The collection proxy shouldn't be loaded and should have the 'Exclude' checkbox checked.
+     * This functionality is designed to simplify the management of Live Update resources.
+     *
+     * @name collectionproxy.set_collection
+     * @param [url] [type:string|hash|url] the collection proxy component
+     * @param [prototype] [type:string|nil] the path to the new collection, or `nil`
+     * @return success [type:boolean] collection change was successful
+     * @return code [type:number] one of the collectionproxy.RESULT_* codes if unsuccessful
+     *
+     * The example assume the script belongs to an instance with collection-proxy-component with id "proxy".
```

Comment:
I don't know if it matters, but in other places we mark examples area with @examples